src/components/data/EntitiesContent.js

```diff
@@ -359,6 +359,87 @@ const EntitiesContent = ({
     ])])
   }
 
+  const entitiesSelected = !_.isEmpty(selectedEntities)
+  const canEdit = !Utils.editWorkspaceError(workspace)
+
+  const renderEditMenu = () => {
+    return !snapshotName && h(MenuTrigger, {
+      side: 'bottom',
+      closeOnClick: true,
+      content: h(Fragment, [
+        h(MenuButton, {
+          onClick: () => setDeletingEntities(true)
+        }, 'Delete')
+      ])
+    }, [h(ButtonSecondary, {
+      disabled: !canEdit || !entitiesSelected,
+      tooltip: Utils.cond(
+        [!canEdit, () => 'You do not have permission to modify this workspace'],
```

Comment:
Nit: I think it's good practice to end sentences with a full stop.
```suggestion
        [!canEdit, () => 'You do not have permission to modify this workspace.'],
```

Reply:
A `git grep "tooltip"` shows that we are very inconsistent about punctuation in tooltips. It would be nice to clarify a guideline for this with UX.

Reply:
Good idea! Brough it up [on Slack](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1648570756312039).

Reply:
Done in [6f43ea4](https://github.com/DataBiosphere/terra-ui/pull/2907/commits/6f43ea4fa36db08e5d4b9265f5d3dd7a95f2a272).